ush/ioda/bufr2ioda/bufr2ioda_satwind_scat.py

```diff
@@ -299,6 +303,12 @@ def bufr_to_ioda(config, logger):
                 .write_attr('long_name', 'Height of Observation') \
                 .write_data(height2)
 
+            # Station Elevation 
```

Comment:
I think the coding norms are complaining here?

Reply:
I fixed the coding norm error.